data-showcase/src/main/groovy/nl/thehyve/datashowcase/search/Operator.groovy

```diff
@@ -0,0 +1,46 @@
+package nl.thehyve.datashowcase.search
+
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+import org.springframework.stereotype.Component
+
+/**
+ * Operator types supported by the criteria builder.
+ */
+@CompileStatic
+@Component
```

Comment:
Why Component? Is it injected as bean somewhere?

Reply:
I'm using it in SearchCriteriaBuilder. Without this annotation I had "not found" exception. Is there a better way to make sure it will be scanned?